src/main/java/me/ryanhamshire/GriefPrevention/Claim.java

```diff
@@ -95,9 +103,7 @@
     //administrative claims are created and maintained by players with the griefprevention.adminclaims permission.
     public boolean isAdminClaim()
     {
-        if (this.parent != null) return this.parent.isAdminClaim();
-
-        return (this.ownerID == null);
+        return this.getOwnerID() == null;
```

Comment:
checking root claim is no longer needed?

Reply:
Sorry for a double reply - GH seems to inline with previous comments on the line but not then copy down the chain. Didn't want it to look like I just blindly resolved this. getOwnerId fetches parent if null